### PR TITLE
Vk editorial edits to ds chapter

### PR DESCRIPTION
**Pull request by drvinceknight**

### CODE REVIEW

File: src/chapters/05/main.tex [hunk 1]
```diff
@@ -166,12 +170,12 @@ \section{Solving with Python}\label{sec:solving-with-python}
 (-0.255, 0.21, 0.045)
 \end{pyout}
 
-we would expect the number of susceptible individuals to reduce by around 0.255
+this means thati the number of susceptible individuals is expected to reduce by
```
Comment (geraintpalmer) [inline src/chapters/05/main.tex:173]: Typo:
```suggestion
this means that the number of susceptible individuals is expected to reduce by
```

File: src/chapters/05/main.tex [hunk 2]
```diff
@@ -253,13 +258,13 @@ \section{Solving with Python}\label{sec:solving-with-python}
...
-We observe that the number of infected individuals increases quickly, and in
+The number of infected individuals increases quickly, and in
 fact the rate of change increases as more individuals are infected. However
 this growth slows down as there are fewer susceptible individuals to infect. Due
 to the equal birth and death rates the overall population size remains constant;
-but we also see after some time period (around 300 time units) the levels of
-susceptible, infected, and recovered individuals becomes seemingly steady, and
-the illness becomes endemic. We can estimate once this steadiness occurs, around
+but after some time period (around 300 time units) the levels of
+susceptible, infected, and recovered individuals stabilise, and
```
Comment (geraintpalmer) [inline src/chapters/05/main.tex:266]: `seems to stabilise`?
We don't know if it does.

File: src/chapters/05/main.tex [hunk 3]
```diff
@@ -415,10 +418,10 @@ \section{Solving with R}\label{sec:solving-with-R}
 }
 \end{Rin}
 
-Using this function returns the instantaneous rate of change for each of the
-three stocks, $S$, $I$ and $R$. If we begin at time 0.0, with 4 susceptible
+This function returns the instantaneous rate of change for each of the
+three stocks, $S$, $I$ and $R$. Starting at time 0.0, with 4 susceptible
```
Comment (geraintpalmer) [inline src/chapters/05/main.tex:422]: Sotcks? Quantities? Functions? We should stick to one.

File: src/chapters/05/main.tex [hunk 4]
```diff
@@ -415,10 +418,10 @@ \section{Solving with R}\label{sec:solving-with-R}
...
 individuals, 1 infected individual, 0 recovered individuals, a vaccine rate
-of 50\% and a birth rate of 0.01, then:
+of 50\% and a birth rate of 0.01, gives::
```
Comment (geraintpalmer) [inline src/chapters/05/main.tex:424]:
```suggestion
of 50\% and a birth rate of 0.01, gives:
```

**Comment by drvinceknight** [comment]
Agreed to revert a80a22c

File: src/chapters/05/main.tex (hunk 2 above)
Comment (drvinceknight) [reply src/chapters/05/main.tex:266]: Not in the sense of a theoretic steady state no (I agree) but in terms of what we can see? This modification was meant more as a language modification (I just felt it read better).

File: src/chapters/05/main.tex (hunk 3 above)
Comment (drvinceknight) [reply src/chapters/05/main.tex:422]: I like quantities.

File: src/chapters/05/main.tex (hunk 4 above)
Comment (drvinceknight) [reply src/chapters/05/main.tex]: Fixed.

File: src/chapters/05/main.tex (hunk 1 above)
Comment (drvinceknight) [reply src/chapters/05/main.tex]: Fixed.

File: src/chapters/05/main.tex (hunk 3 above)
Comment (geraintpalmer) [reply src/chapters/05/main.tex:422]: :+1: I'll go through and change them all to quantiles. I might in the theory section say they are also sometimes called stocks (I use a stock and flow diagram), and that they are essentially functions

**Pull request merged**